### PR TITLE
Add dingo fact

### PR DESCRIPTION
**Pull request by Erovia**
It's most important this gets merged.

### CODE REVIEW

**Review by tzarc — APPROVED** [review]
I can only hit approve once, sadly.

File: src/potato-facts.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default [
   'Potatoes are totally gluten-free.',
   'One of the most basic Incan measurements of time was the time it took to cook a potato.',
   'If a potato is exposed to light while growing, it will turn green and become poisonous.',
+  'F**k mate, that\'s drier than a dead dingo\'s donger.'
```
Comment (noroadsleft) [inline src/potato-facts.js:17]:
```suggestion
  'F**k mate, that\'s drier than a dead dingo\'s donger.',
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: src/potato-facts.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ export default [
   'Potatoes are totally gluten-free.',
   'One of the most basic Incan measurements of time was the time it took to cook a potato.',
   'If a potato is exposed to light while growing, it will turn green and become poisonous.',
+  'F**k mate, that\'s drier than a dead dingo\'s donger.',
```
Comment (tzarc) [inline src/potato-facts.js:17]:
```suggestion
  'F**k mate, that potato\'s drier than a dead dingo\'s donger.',
```

**Review by tzarc — CHANGES_REQUESTED** [review]

**Review by yanfali — APPROVED** [review]

**Comment by yanfali** [comment]
@Erovia rebase this on latest master so it'll build again. We stopped using travis.

**Comment by yanfali** [comment]
closing

**Pull request closed without merging**